CHANGELOG.rst

```diff
@@ -9,7 +9,7 @@ Next Release (TBD)
   redeployments of chalice applications and in the CloudFormation template
   generated by ``chalice package``
   (`#339 <https://github.com/awslabs/chalice/issues/339>`__)
-* Fix autogen_policy in config being ignored
+* Fix `autogen_policy` in config being ignored
```

Comment:
Oh it's rst not markdown. That gets me as well. Probably want to use `` instead of `